aws/ami/scylla.json

```diff
@@ -80,6 +80,12 @@
         "sudo {{user `python`}} /home/{{user `ssh_username`}}/scylla_install_ami {{ user `install_args` }}"
       ],
       "type": "shell"
+    },
+    {
+      "source": "/home/{{user `ssh_username`}}/scylla-packages.txt",
```

Comment:
why are you downloading the file to the build machine?

Reply:
Please see description at https://github.com/scylladb/scylla-pkg/issues/1869, this intended to upload `scylla-packages.txt` to S3 (somewhere at downloads.scylladb.com), after finishing build of the AMI on Jenkins job.
The uploading part will be implement on Jenkins job after succeeded building the AMI, not this script.
So we need to do here is moving generated `scylla-packages.txt` from temporarily created AMI instance to the Jenkins node.